conjur/cli.py

```diff
@@ -339,48 +367,75 @@ def handle_logout_logic(cls, ssl_verify=True):
-                                         list_logic=list_logic,
+        list_controller = ListController(list_logic=list_logic,
                                          list_data=list_data)
         list_controller.load()
 
     @classmethod
-    def handle_variable_logic(cls, ssl_verify=True, args=None, client=None):
+    def handle_variable_logic(cls, args=None, client=None):
         """
         Method that wraps the variable call logic
         """
         variable_logic = VariableLogic(client)
         if args.action == 'get':
             variable_data = VariableData(action=args.action, id=args.id, value=None,
                                          variable_version=args.version)
-            variable_controller = VariableController(ssl_verify=ssl_verify,
-                                                     variable_logic=variable_logic,
+            variable_controller = VariableController(variable_logic=variable_logic,
                                                      variable_data=variable_data)
             variable_controller.get_variable()
         elif args.action == 'set':
             variable_data = VariableData(action=args.action, id=args.id, value=args.value,
                                          variable_version=None)
-            variable_controller = VariableController(ssl_verify=ssl_verify,
-                                                     variable_logic=variable_logic,
+            variable_controller = VariableController(variable_logic=variable_logic,
                                                      variable_data=variable_data)
             variable_controller.set_variable()
 
     @classmethod
-    def handle_policy_logic(cls, ssl_verify=True, policy_data=None, client=None):
+    def handle_policy_logic(cls, policy_data=None, client=None):
         """
         Method that wraps the variable call logic
         """
         policy_logic = PolicyLogic(client)
-        policy_controller = PolicyController(ssl_verify=ssl_verify,
-                                             policy_logic=policy_logic,
+        policy_controller = PolicyController(policy_logic=policy_logic,
                                              policy_data=policy_data)
         policy_controller.load()
 
+    @classmethod
+    def handle_user_logic(cls, args=None, client=None, resource=None):
+        """
+        Method that wraps the user call logic
+        """
+        credentials = CredentialsFromFile()
+        user_logic = UserLogic(ConjurrcData, credentials, client, resource)
+        if args.action == 'rotate-api-key':
+            user_resource_data = UserResourceData(action=args.action,
+                                                  id=args.id,
+                                                  new_password=None)
+            user_controller = UserController(user_logic=user_logic,
+                                             user_resource_data=user_resource_data)
+            user_controller.rotate_api_key()
```

Comment:
better to extract to private function.. same for other else_ifs here

Reply:
btw if you think it will make the class noisy we can create class handle_user_logic. will look like this

```suggestion
class handle_user_logic:
    def __init__():
         credentials = CredentialsFromFile()
        user_logic = UserLogic(ConjurrcData, credentials, client, resource)
    def rotate_api_key():
        user_resource_data = UserResourceData(action=args.action,
                                                  id=args.id,
                                                  new_password=None)
            user_controller = UserController(user_logic=user_logic,
                                             user_resource_data=user_resource_data)
            user_controller.rotate_api_key()
       def change_password():
           user_resource_data = UserResourceData(action=args.action,
                                                  id=None,
                                                  new_password=args.password)
            user_controller = UserController(user_logic=user_logic,
                                             user_resource_data=user_resource_data)
            user_controller.change_password()
       
```

Reply:
this would then mean I would need to do this for all other handle_x_logic in this CLI class which is not what I want to get into now. I will open a card for a better design